Adding Telemetry info for Analytics-Swift and Analytics-Kotlin SDKs (#7184)
segmentio/segment-docs: src/connections/sources/catalog/libraries/mobile/kotlin-android/index.md

> warning ""
> If you are using the Analytics Android (Classic) SDK, you can find [the documentation here](/docs/connections/sources/catalog/libraries/mobile/android). Many of the features available in the Analytics Kotlin SDK are not available in the Analytics Android (Classic) SDK.

## Telemetry

> Telemetry data related to the configuration and usage of the Analytics-Kotlin SDK is collected and sent to Segment by default.  This includes some basic configuration information, plugins and event types used, and basic error information.  No PII or event data is collected and the information is downsampled to reduce traffic.

Reviewer: ```suggestion
Telemetry data related to the configuration and usage of the Analytics-Kotlin SDK is collected and sent to Segment by default.  This includes some basic configuration information, plugins and event types used, and basic error information.  Segment downsamples the data to minimize traffic and doesn't collect any personally identifiable information (PII) or event data.
```

> The Telemetry system can be disabled at any time by setting `Telemetry.enable = false`.

Reviewer: ```suggestion
You can disable telemetry at any time by setting `Telemetry.shared.enable = false`.
```

> When internal errors or errors from plugins occur, the writekey may be included with the error data to help customers track down problems.  This can be disabled by setting `Telemetry.sendWriteKeyOnError = false`.

Reviewer: ```suggestion
When internal errors or errors from plugins occur, the writekey may be included with the error data to help customers track down problems.  You can disable this by setting `Telemetry.sendWriteKeyOnError = false`.
```
